react-quill/app/components/textEditor.client.tsx

```diff
@@ -0,0 +1,48 @@
+import ReactQuill from "react-quill";
+
+interface props {
+  theme: string;
+  placeholder: string;
+  modules: object;
+  onChange?: (...args: any) => any;
+  name?: string;
+  value: string;
+}
```

Comment:
Let's make it clear we're just passing these props

```suggestion
type ReactQuillProps = ComponentProps<typeof ReactQuill>;
type Props = Pick<ReactQuillProps, 'name' | 'onChange' | 'placeholder' | 'theme' | 'value'>;
```